torch/_higher_order_ops/while_loop.py

```diff
@@ -6,6 +6,7 @@
 from torch._higher_order_ops.utils import (
     _has_potential_branch_input_alias,
     _has_potential_branch_input_mutation,
+    _is_compiling_hoo,
```

Comment:
should this be hook?

Reply:
Should be `hoo` :) It stands for "higher-order op", I saw this abbreviation used here and there.